libs/sdk-ui-charts/src/highcharts/chartTypes/_chartCreators/getChartOrientationConfiguration.ts

```diff
@@ -1,10 +1,29 @@
 // (C) 2023 GoodData Corporation
 
 import { IChartConfig } from "../../../interfaces/index.js";
-import { HighchartsOptions } from "../../lib/index.js";
+import { HighchartsOptions, XAxisOptions } from "../../lib/index.js";
 import { IChartOptions } from "../../typings/unsafe.js";
 import { isWaterfall } from "../_util/common.js";
 
+function shortenXAxisLabel(xAxis: XAxisOptions[]) {
+    if (!xAxis?.[0]?.categories || xAxis[0].categories.findIndex((item) => item.length >= 50) < 0) {
```

Comment:
Use Array.every and bellow condition It will by much readable
https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/every

Reply:
Thank you for your suggestion, the `every` function will go through all elements of an array. But we need if there is one item that has a length larger than 50, we should add the ellipsis for the xAxis. So the `findIndex` function is more suitable because it will return the first item index match with the conditional. WDYT?

Reply:
I see, .some would be better

Reply:
I have updated this. Thank you!